src/sponsorblock.js

```diff
@@ -403,12 +411,8 @@ window.addEventListener(
         window.sponsorblock = null;
       }
 
-      if (configRead('enableSponsorBlock')) {
-        window.sponsorblock = new SponsorBlockHandler(videoID);
-        window.sponsorblock.init();
-      } else {
-        console.info('SponsorBlock disabled, not loading');
-      }
+      window.sponsorblock = new SponsorBlockHandler(videoID);
+      window.sponsorblock.init();
```

Comment:
Why is the SponsorBlock object always created now?

Reply:
Oh, I think I see. It's always running, but if it's not enabled, `getSkippableCategories()` just returns an empty list.